src/calculateRentalCost.js

```diff
@@ -29,6 +29,27 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+  const total = cost * days;
+  const largeDiscount = 50;
+  const smallDiscount = 20;
+  const noDiscount = 2;
+  const largeDiscountReq = 7;
+  const smallDiscountReq = 3;
+
+  if (days <= noDiscount) {
+    return total;
+  }
+
+  if (days < largeDiscountReq) {
+    return total - smallDiscount;
+  }
+
+  if (days >= smallDiscountReq) {
+    return total - largeDiscount;
+  }
+
+  return total;
```

Comment:
What if you checked from the highest condition to the lowest? You would avoid 1 if statement, and you wouldn't have to write `return total` twice

Reply:
I was thinking the very same when I wrote it but wasn't sure.